test/Dapr.Actors.Test/Serialization/ActorIdJsonConverterTest.cs

```diff
@@ -62,11 +62,10 @@ public void CanDeserializeActorId()
         [Fact]
         public void CanDeserializeNullActorId()
         {
-            var id = ActorId.CreateRandom().GetId();
             var document = $@"
```

Comment:
No need for $ as there's no interpolation happening

Reply:
@WhitWaldo Solved in my latest commit.